filters/diag/weakrand.go

```diff
@@ -0,0 +1,46 @@
+package diag
+
+import (
+	"math/rand"
+	"sync"
+)
+
+type reader struct {
+	c  []byte
+	i  int
+	mx *sync.Mutex
+}
+
+var randomChars = []byte("ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789")
+
+func weakRandom() *reader {
```

Comment:
It is really weak, better not ask for more than 62 characters :)

Reply:
:smile: 

yeah.. So the thing is that we don't want to turn Skipper into a real random generator, we just want to generate some data that is not too predicable for diagnostic purposes. https://xkcd.com/221/